oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/util/Utils.java

```diff
@@ -1009,6 +1009,16 @@ public static boolean isEmbeddedVerificationEnabled(final DocumentNodeStoreBuild
         return builder.isEmbeddedVerificationEnabled() || (docStoreEmbeddedVerificationFeature != null && docStoreEmbeddedVerificationFeature.isEnabled());
     }
 
+    /**
+     * Returns the full GC mode value for the DocumentNodeStore.
+     *
+     * @param builder instance for DocumentNodeStoreBuilder
+     * @return true if full GC is enabled else false
+     */
+    public static int getFullGCMode(final DocumentNodeStoreBuilder<?> builder) {
```

Comment:
This utility method doesn't make sense, it only returns `fullGcMode` from the `builder` object. I would remove this.

Reply:
Fixed.